### ISSUE TITLE
Circle of the Winds PQ Stage 1 bug

### ISSUE THREAD

**Issue by LadyCereus** [WarEmu/WarBugs#14063]
Hi - the first stage of the Circle of the Winds PQ in Saphery seems bugged.  You need to click 8 stones, but only one of the stones gives you credit.  I was able to complete the PQ by waiting for that one stone to respawn 8 times, but that's the only way it would move on to the next stage.  Thanks for your help!

**Comment by Kabucoror** [comment]
4/8 are counting now, the rest is not possible atm.

**Comment by Ootootoo** [comment]
Can confirm still persist
![Aduril_003](https://user-images.githubusercontent.com/50114048/72743716-1ae34a00-3ba4-11ea-8976-322d305d69e3.jpg)

**Comment by Delegator2** [comment]
Yup just ran into this today, only the stone of Azyr gives credit. It would be silly to just click on 8 stones and get stage 1 done, but clicking on one stone 8 times is just as silly. Seems like something is really missing here.
